README.md

```diff
@@ -1,9 +1,6 @@
 # Wolox on Rails - Requests
 [![Gem Version](https://badge.fury.io/rb/wor-requests.svg)](https://badge.fury.io/rb/wor-requests)
-[![Dependency Status](https://gemnasium.com/badges/github.com/Wolox/wor-requests.svg)](https://gemnasium.com/github.com/Wolox/wor-requests)
-[![Build Status](https://travis-ci.org/Wolox/wor-requests.svg)](https://travis-ci.org/Wolox/wor-requests)
```

Comment:
What happend here? I mean, I already saw that the image is not displaying... We know why happend that?

Reply:
- Gemnasium https://docs.gitlab.com/ee/user/project/import/gemnasium.html
- Travis ci, my error
- Test Coverage: https://github.com/codeclimate/ruby-test-reporter, don't report a test coverage